p/themes/base-theme/template.rtl.css

```diff
@@ -1418,13 +1418,28 @@ br {
 }
 
 #close-slider.active {
+	background: rgba(0, 0, 0, 0.2);
 	right: 0;
 }
 
 #close-slider img {
 	display: none;
 }
 
+#close-slider.active img {
+	padding: 0.5rem;
+	display: inline-block;
+	position: absolute;
+	top: 0.5rem;
+	right: 0.5rem;
+	filter: grayscale(100%) brightness(2.5);
+}
+
+#close-slider.active:hover img {
+	background: rgba(0, 0, 0, 0.2);
+	border-radius: 50%;
+}
+
```

Comment:
This and the whole X strikes me as odd, making it both disconnected and as if you have to click on that X.

Reply:
I changed it to the default behavior like the other close areas

Reply:
And fixed the CSS.